e2e-tests/authenticated/viewAsset.spec.ts

```diff
@@ -150,3 +150,33 @@ test('Show map settings', async ({ page }) => {
 		path: `./test-session/map-settings.png`,
 	})
 })
+
+test('Map should be vertically resizable', async ({ page }) => {
+	const mapContainer = page.locator('#map-container')
+	const resizeBtn = page.locator('#map-container-resize')
+
+	if (resizeBtn !== undefined && mapContainer !== undefined) {
```

Comment:
**problem:** [Locators](https://playwright.dev/docs/api/class-page#page-locator) will never be `undefined`, because they do not get evaluated immediately. Therefore, combine it with an action, e.g. assert that they are actually visible.
```
await expect(locator).toBeVisible()
```